cli/docker_image/show.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+
+import xml.etree.ElementTree as ET
+
+
+def configure_parser(docker_image_subparser):
+    docker_image_show = docker_image_subparser.add_parser(
+        "show", help="Show informations about a docker image"
+    )
+    docker_image_show_subparser = docker_image_show.add_subparsers(
+        dest="information", help="Information to show"
+    )
+    docker_image_show_subparser.required = True
+
+    init_show_vulnerabilities_subcommand(docker_image_show_subparser)
+
+
+def subcommand(args, api):
+    if args.information == "vulnerabilities":
+        vulnerabilities_subcommand(args, api)
+    else:
+        print(f"'{args.object}'")
+
+
+# Vulnerabilities
+
+
+def init_show_vulnerabilities_subcommand(docker_image_show_subparser):
+    docker_image_show_vulnerabilities_parser = docker_image_show_subparser.add_parser(
+        "vulnerabilities", help="Show vulnerabilities about a scanned docker image"
+    )
+    docker_image_show_vulnerabilities_parser.add_argument("docker_image_id")
+    docker_image_show_vulnerabilities_parser.add_argument(
+        "--format",
+        help="Output format. The only supported format is 'xml:filename.xml'.",
+    )
+
+
+def vulnerabilities_subcommand(args, api):
+    docker_image = api.docker_image(args.docker_image_id)
+    server_id = str(docker_image[6])
+    server = api.server(server_id)
+
+    cve_announcements = []
+    vulnerabilities = server.cve_announcements
+    for vulnerability in vulnerabilities:
+        cve_code = vulnerability.cve_code
+        cve_announcements.append(api.cve_announcement(cve_code))
```

Comment:
Here we retrieve information about the CVE required in the output. As it is not possible to retrieve a group of CVE in one request, we need to do a request per CVE. This can slow the command if there is a lot of vulnerabilities.